src/index.html

```diff
@@ -22,6 +22,408 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-moyo"
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__item-button"
+              href="/"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              chref="/"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="#"
```

Comment:
The 'alt' attribute for the image is set to '#', which is not descriptive. Consider providing a meaningful description for accessibility purposes.